plugins/modules/dcnm_policy.py

```diff
@@ -994,18 +997,40 @@ def dcnm_policy_send_message_to_dcnm(self):
         # Once all policies are deleted, do a save & deploy so that the deleted policies are removed from the
         # switch
         if (snos != []) and (mark_delete_flag is True):
-            resp = self.dcnm_policy_save_and_deploy(snos)
-            if (
-                resp
-                and (resp["RETURN_CODE"] != 200)
-            ):
-                self.module.fail_json(msg=resp)
+
+            retries = 0
+            while retries < 50:
+
+                retries += 1
+                resp = self.dcnm_policy_save_and_deploy(snos)
+                if (
+                    resp
+                    and (resp["RETURN_CODE"] != 200)
+                ):
+                    self.module.fail_json(msg=resp)
+
+                path = self.paths["CONFIG_PREVIEW"].format(self.fabric)
```

Comment:
What does `CONFIG_PREVIEW` indicate?  Can you add some comments here?

Reply:
It gets the current status of the confguration on the DCNM. If it is not "In-Sync" then there re more commands to be deployed. So we keep retrying

Reply:
Added comments for this